assets/app/features/upgrade/controllers/upgrade-backup.controller.js

```diff
@@ -68,18 +75,9 @@ function() {
                             // Determine the content type from the header
                             contentType = headers['content-type'],
 
-                            backupBlob = new Blob([response.data], {type: contentType}),
-                            backupObjectUrl = URL.createObjectURL(backupBlob),
+                            backupBlob = new Blob([response.data], {type: contentType});
```

Comment:
Depending on the portability requirements (i.e. how old browsers need to be supported), we can make it even simpler: if you set `responseType: 'blob'` in `crowbar-backup.factory.js` you will get `response.data` already in blob format and you can drop the whole manual `Blob` object creation.

`Blob` has pretty decent support in modern desktop browsers: https://developer.mozilla.org/en-US/docs/Web/API/Blob#Browser_compatibility